app/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,10 @@
     <string name="singlePlayerButton_text">SINGLE-PLAYER GAME</string>
     <string name="multiPlayerButton_text">MULTI-PLAYER GAME</string>
     <string name="highScoreButton_text">HIGH SCORES</string>
+    <string name="titleSingleDefeatTextView_text">YOU LOST!</string>
+    <string name="titleCongratulationsTextView_text">...and set \n \n the NEW RECORD!</string>
```

Comment:
Prefer margin and padding settings to hard returns when formatting line spacing.